### PR TITLE
Remove optionalTarget: Clean up

### PR DESCRIPTION
**Pull request by Mugen87**
Some clean up to avoid warnings...

### CODE REVIEW

File: src/core/BufferGeometry.js [hunk 1]
```diff
@@ -278,15 +278,21 @@ BufferGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 	center: function () {
 
-		this.computeBoundingBox();
```
Comment (mrdoob) [inline src/core/BufferGeometry.js]: Hmm, this change can break user code if they did this:

```js
var center1 = geometry1.center();
var center2 = geometry2.center();
```

Comment (Mugen87) [reply src/core/BufferGeometry.js]: I see. I guess we have to create a new object in `center()`.

Comment (mrdoob) [reply src/core/BufferGeometry.js]: For now, yeah...

Comment (Mugen87) [reply src/core/BufferGeometry.js]: Okay, changed the code.

**Comment by WestLangley** [comment]
@mrdoob @Mugen87 The problem with the code is `.center()` should return `this`. The method is not `.getCenter()`.

I suggest making that change, even though it is breaking, because the code is incorrect. Then the changes @Mugen87 made previously are fine.

**Comment by Mugen87** [comment]
> I suggest making that change, even though it is breaking, because the code is incorrect.

Yeah, sounds reasonable.

**Comment by Mugen87** [comment]
I've only found one example (`css3d_molecules.html`) where the return value of `center()` was used.

**Comment by Mugen87** [comment]
BTW: `.center()` does not actually return the center but the offset that is used to translate the geometry to the center of the AABB. That's weird and i think the change clarifies the method.

File: examples/css3d_molecules.html [hunk 2]
```diff
@@ -334,8 +334,8 @@
 					var geometryBonds = pdb.geometryBonds;
 					var json = pdb.json;
 
-					var offset = geometryAtoms.center();
-					geometryBonds.translate( offset.x, offset.y, offset.z );
+					geometryAtoms.center();
+					geometryBonds.center();
```
Comment (WestLangley) [inline examples/css3d_molecules.html]: Does that produce the same offset for both atoms and bonds?

Comment (Mugen87) [reply examples/css3d_molecules.html]: The visual result is the same.

BTW: I've missed `webgl_loader_pdb`. Applied the same change there, too.

Comment (WestLangley) [reply examples/css3d_molecules.html]: Sorry. How you can center the atoms and bonds separately makes no sense to me.

**Comment by WestLangley** [comment]
Please file API changes in a separate PR.

File: examples/css3d_molecules.html (hunk 2 above)
Comment (Mugen87) [reply examples/css3d_molecules.html]: I guess that works because `PDBLoader` builds both geometries from the same vertex data.

**Comment by Mugen87** [comment]
Okay, i undo the change to `.center()` in `Geometry` and  `BufferGeometry`.

File: examples/css3d_molecules.html (hunk 2 above)
Comment (WestLangley) [reply examples/css3d_molecules.html]: I would inline the `center()` code, if you have to, and not rely on that.

**Comment by WestLangley** [comment]
Thanks... Not to derail, but do you know how to collapse these commits into a single 'Remove optionalTarget: clean up' commit, so all the intermediary changes -- and reverted changes -- do not appear in the file commit history?

**Comment by Mugen87** [comment]
Okay, i made a force push. The old commits are now gone. I also inlined the code in the PDB examples. Hope, it's now okay like that :sweat_smile:

File: examples/css3d_molecules.html (hunk 2 above)
Comment (Mugen87) [reply examples/css3d_molecules.html]: Okay :innocent:

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**